### DISCUSSION TITLE
Crash when deleting IPM.Configuration.Autocomplete

### DISCUSSION THREAD

**Discussion by kilobajt** [microsoft/mfcmapi#889] [General]
MPCMAPI crashes when deleting IPM.Configuration.Autocomplete. This happens to us on newest Outlook 365 on all 6 computers and on both x86 and x64 Outlook editions (depending on computer). We can't import older autocomplete lists and it's really annoying. We are using same steps as tonyc2000 in this post:
https://github.com/stephenegriffin/mfcmapi/issues/106

Is there any other way to import autocomplete list? We really need it.

Crash info:
Napačno ime programa: MFCMAPI.exe, različica: 19.2.19058.831, časovni žig: 0x5c75fa9b
Napačno ime modula: pstprx32.dll, različica: 16.0.11231.20120, časovni žig: 0x5c48dc13
Koda izjeme: 0xc0000005
Napačen odmik: 0x00000000001461d0
Napačen id procesa: 0x3954
Napačen začetni čas programa: 0x01d4ce8c80a04caa
Napačna programska pot: C:\Users\Borut\Downloads\MFCMAPI.exe.x64.19.2.19058.831\MFCMAPI.exe
Napačna pot modula: C:\Program Files\Microsoft Office\Root\Office16\pstprx32.dll
Id poročila: 815f6a0b-6f00-4e16-88a7-f459a63fcb4b
Napačno polno ime paketa:
Napačen ID programa, sorodnega paketu:

**Comment by stephenegriffin** [comment]
Unfortunately, all I can tell from this is that it's a crash in pstprx32.dll, an Outlook module not under MFCMAPI's control. Perhaps with the AppCrash details (as reported in #106) I might be able to look up a Watson bucket and pull up a stack.

**Comment by stephenegriffin** [comment]
@kilobajt or @tonyc2000 If there's any chance you can get a procdump I'd be happy to look at that as well.

**Comment by kilobajt** [comment]
@stephenegriffin Finally got it. It's here: http://www.kilobajt.net/MFCMAPI.exe_190326_102303.dmp

**Comment by k9gardner** [comment]
This is still crashing. Note: when copying to new computer it's not really an issue having two autocomplete files in place, as the default file is theoretically empty.

**Comment by sonextc** [comment]
Hello,
it is still crashing in the moment of deleting message. Application MFCMAPI completely crash.
It would be really great to help us. I tried the above mentioned DMP file, but it did not help us.

Thank you very much.

**Comment by stephenm93** [comment]
Not sure if this issue was ever solved, but I discovered a solution:
1) Open the associated contents table of the destination "Inbox" for the new auto complete data
2) DO NOT DELETE the existing IPM.Configuration.AutoComplete message (note the ConversationID referenced)
3) Proceed with Folder -> Import of the new auto complete MSG
4) Refresh the View and you will have two (2) IPM.Configuration.AutoComplete (now with different Conversation ID)
5) select the "old" IPM.Configuration.AutoComplete records, right click and Open Message; find each Property with a value of IPM.Congifuration.AutoComplete and delete the property.  If a property fails to delete, skip and proceed to next step.
6) Close the message and select the "new" IPM.Configuration.Autocomplete record and Open Message; find each Property with a value of IPM.Configuration.AutoComplete; righ click and Copy Property.
7) Close the message and re-open the "old" from step 5.  Right click and paste the property.  Repeat for each property in step 6.
8) from Step 5, if a property failed to delete, find the identical property name within the "new" message, right click - Edit Property and copy the Binary string.
9) Close the message and re-open the identical Property in the "old" message record, right click to Edit the property and paste the binary data.
10) Restart Outlook and the auto complete should now work.  It worked for me.  :)

**Comment by stephenegriffin** [comment]
got a look at the dump - best I can offer is this Outlook client hasn't been updated in over a year.

**Comment by fish7221** [comment]
BIG THANKS to stephenm93 tip which helped me solve the problem by deleting and copying the IPM.Configuration.Autocomplete records from the new MSG file

**Comment by stephen-mca** [comment]
Thanks also to stephennm93, deleting and copying the individual records worked for me too. And after Outlook restarted, I noticed there is now only one IPM.Configuration.Autocomplete. I am guessing Outlook client or something deleted the duplicate when starting.

**Comment by jcamacho0505** [comment]
HUUge thanks to **stephenm93** I normally do not post comments or reviews...unless I'm complaining about something, but i went through the Hassel of creating an account just to say THANKS!

**Comment by HughWarrington** [comment]
I followed the steps by @stephenm93 but in step 7 the 'paste property' option was greyed out. So in effect I just skipped steps 6-9. However when I next started Outlook, my autocomplete list was populated from the imported file. Perhaps there are now missing or duplicate entries from the 'old' autocomplete list, but that does not matter to me. I then closed Outlook again, and afterwards in MFCMAPI I could see only single IPM.Configuration.AutoComplete message. Perhaps Outlook did the right thing and deleted the old message.

**Comment by si458** [comment]
i just want to also thank @stephenm93 and @HughWarrington for the help!
the bos got a new machine and was 110% angry that all his autocomplete had vanished! (we use imap not exchange/365)
steps are indeed 1-5 from @stephenm93 
then just close the app and open outlook and away it went!
just like @HughWarrington suggested!
you guys saved my bacon literally!!!
